apps/sms/services/js/message_manager.js

```diff
@@ -91,16 +135,20 @@ var MessageManager = {
     }
   },
 
+  /**
+   * Function to obtain all the Threads and perform a function 
```

Comment:
Also, there are other functions also, other than applying on each thread. So, better to make it general. 
Something like this: 'Method to obtain all the Threads and takes a set of callback functions as options parameter'